src/main/java/com/example/WMS/entity/ProductDetails.java

```diff
@@ -0,0 +1,16 @@
+package com.example.WMS.entity;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@Builder
+@ToString
+public class ProductDetails {
```

Comment:
Maintain more feilds like created date and last Updated Date

Reply:
Maintaining created date, last ordered date and last incoming goods date to product details